### DISCUSSION TITLE
Vertico fails to update options after typing invalid string and then deleting backwards

### DISCUSSION THREAD

**Discussion by NashRickert** [minad/vertico#449] [Q&A] [answered]
Hello,

I had a question about vertico and its configuration. Basically when I use vertico, if I type a command that doesn't actually exist, vertico displays the most recent option(s) and shows error: "Invalid completion style orderless". When I delete characters, instead of expanding to the options previously available for the current state of the minibuffer, it keeps displaying the last valid option before there were no options left. So for example, if I type "eval-bufj" and get the error and then delete back to "eva", vertico won't update the options and will continue to display "eval-buffer" as the only valid choice. I was wondering if there was anything I could do to configure vertico so it continues to update even after commands with no matches are typed in

Thank you

**Comment by minad** [comment, marked as answer]
It seems you've configured the `orderless` completion style but you did not install the Orderless package from ELPA/MELPA. Try to install it with `M-x package-install`.
